.env

```
@@ -0,0 +1,4 @@
+PRIVATE_KEY=011d4ba8e7b549ee9cd718df4c25abed
+RPC_URL_SEPOLIA=https://eth-mainnet.g.alchemy.com/v2/MBGDHRRT3_qjtqKzM2DZk
+RPC_URL_FUJI=https://avax-fuji.g.alchemy.com/v2/MBGDHRRT3_qjtqKzM2DZk
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The RPC_URL_FUJI key should go before the RPC_URL_SEPOLIA key

(UnorderedKey)